src/main/java/school/hei/haapi/model/User.java

```diff
@@ -85,21 +89,34 @@ public class User implements Serializable {
 
   private String profilePictureKey;
 
+  // RELATION (TEACHER): Awarded Courses
   @OneToMany(fetch = FetchType.LAZY, mappedBy = "mainTeacher")
   @ToString.Exclude
   private List<AwardedCourse> awardedCourses;
 
+  // RELATION (STUDENT): Group Flows
   @OneToMany(mappedBy = "student", fetch = LAZY)
   @ToString.Exclude
   private List<GroupFlow> groupFlows;
 
+  // RELATION (STUDENT): Grades
   @OneToMany(mappedBy = "student", fetch = LAZY)
   @ToString.Exclude
   private List<Grade> grades;
 
+  // RELATION (STUDENT): Work Documents
   @OneToMany(mappedBy = "student", fetch = LAZY)
   private List<WorkDocument> workDocuments;
 
+  // RELATION (MONITOR - STUDENT): Which Monitor follows which students or which student is
+  // following by which monitor
+  @ManyToMany(fetch = LAZY)
```

Comment:
lazy fetch is done by default

Reply:
this attribute only shows students followed by the monitor. It does not contain the monitor himself